src/hint-core.scss

```diff
@@ -34,13 +34,15 @@
 		@include vendor('transition-delay', $hideDelay);
 	}
 
-	&:hover:before, &:hover:after {
+	&:hover:before, &:hover:after,
+	&:not(button):focus:before, &:not(button):focus:after {
 		visibility: visible;
 		opacity: 1;
 	}
 
-	&:hover:before, &:hover:after {
-		// $showDelay will apply as soon as element is hovered.
+	&:hover:before, &:hover:after,
+	&:not(button):focus:before, &:not(button):focus:after {
```

Comment:
@PalleZingmark I am not sure if this fixes all issues. Just like `button`, I think an anchor will also grab focus on getting clicked and start showing the tooltip.